db/migrate/20151124183140_remove_talks.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveTalks < ActiveRecord::Migration
   def change
-    drop_table :talks
+    drop_table :talks do
```

Comment:
No, it's here:

http://apidock.com/rails/ActiveRecord/Migration/CommandRecorder/invert_drop_table